vmdb/app/presenters/explorer_presenter.rb

```diff
@@ -105,8 +102,6 @@ def process
 
     @out << "dhxAccord.openItem('#{@options[:open_accord]}');" unless @options[:open_accord].to_s.empty?
 
-    @options[:trees_to_replace].each { |tree, opts| @out << replace_tree(tree,opts) }
```

Comment:
The option is never used (last instance removed in `eb41dac`) -- and is the only reason this class was being handed `@temp`. Seemed simpler to just kill the dead code, rather than introduce needless `instance_variable_get` nastiness.

I could probably make that clearer in the commit message. :sweat_smile: